apps/cookbook/src/app/page/side-nav/side-nav.component.ts

```diff
@@ -87,9 +95,11 @@ export class SideNavComponent implements OnInit {
     }
   }
 
-  onLinksArrowUpDown(event: KeyboardEvent) {
+  onLinksArrowUpDown(event: KeyboardEvent, sideNavGroupKey: string) {
     event.preventDefault();
-    const listElements: HTMLAnchorElement[] = this.componentLinks.map((link) => link.nativeElement);
+    const listElements: HTMLAnchorElement[] = this.sideNavGroups[sideNavGroupKey].map(
```

Comment:
Maybe inline and simpify?
```suggestion
    const listElements: HTMLAnchorElement[] = (sideNavGroupKey === 'resources' ? this.resourceLinks : componentLinks).map(
```

Reply:
Yeah, I was designing a tiny bit for the future here, but fine by me to remove it!